distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -339,9 +339,11 @@ grant codeBase "file:/catalog-core-standardframework/platform-solr-server-standa
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
 }
 
-grant codeBase "file:/catalog-plugin-videothumbnail" {
+grant codeBase "file:/catalog-plugin-videothumbnail/catalog-rest-endpoint/catalog-core-standardframework" {
```

Comment:
❓Seems odd to me that we would need to grant those permissions to the `catalog-rest-endpoint` and `catalog-core-standardframework` bundles here. Could/should we use `AccessController.doPrivileged()` where the `ffmpeg` binary is executed instead to avoid having to add those permissions?